code-of-conduct.md

```diff
@@ -7,10 +7,12 @@ As members of the community,
 - We pledge to welcome those interested in joining the community, and realize that including people with a variety of opinions and backgrounds will only serve to enrich our community. In particular, discussions relating to pros/cons of various technologies, programming languages, and so on are welcome, but these should be done with respect, taking proactive measure to ensure that all participants are heard and feel confident that they can freely express their opinions.
 - We pledge to welcome questions and answer them respectfully, paying particular attention to those new to the community. We pledge to provide respectful criticisms and feedback in forums, especially in discussion threads resulting from code contributions.
 - We pledge to be conscientious of the perceptions of the wider community and to respond to criticism respectfully. We will strive to model behaviors that encourage productive debate and disagreement, both within our community and where we are criticized. We will treat those outside our community with the same respect as people within our community.
-- We pledge to help the entire community follow the code of conduct, and to not remain silent when we see violations of the code of conduct. We will take action when members of our community violate this code such as contacting confidential@astropy.org (all emails sent to this address will be treated with the strictest confidence) or talking privately with the person.
+- We pledge to help the entire community follow the code of conduct, and to not remain silent when we see violations of the code of conduct. We will take action when members of our community violate this code. such as contacting the Ombudsperson at confidential@astropy.org (all emails sent to this address will be treated with the strictest confidence) or talking privately with the person.
```

Comment:
Perhaps start with `talking privately with the person` and follow with sending email to the Ombudsperson?  That also makes the sentence easier to parse since there is this long parenthetical note.  That note can be shortened to `(emails to this adress are treated with strictest confidence)`.